pkg/odo/genericclioptions/context.go

```diff
@@ -118,9 +118,11 @@ func getValidConfig(command *cobra.Command) (*config.LocalConfigInfo, error) {
 		// Gather nessasary info
 		p := command.Parent()
 		r := command.Root()
+		fmt.Println("root is", r.Name(), "parent is", p.Name())
 		afs := FlagValueIfSet(command, ApplicationFlagName)
 		// Find the first child of the command. As some groups are allowed even with non existent config
 		fcc := getFirstChildOfCommand(command)
+		fmt.Println("fcc is", fcc.Name(), "command name is", command.Name())
```

Comment:
I suspect that this is debugging output left accidentally

Reply:
yeah :see_no_evil: